graphistry/dep_manager.py

```diff
@@ -0,0 +1,30 @@
+import importlib
+
+class DepManager:
+    def __init__(self):
+        self.pkgs = {}
+
+    def __getattr__(self, pkg:str):
+        self._add_deps(pkg)
+        try:
+            return self.pkgs[pkg]
+        except KeyError:
+            return None
```

Comment:
instead of hiding the exception from the user, it's probably more informative and unsurprising to:

* cache the exception during `_add_deps`
* fetch the cached exception, instead of None, and rethrow here, instead of return